src/items/tools/knife.js

```diff
@@ -0,0 +1,53 @@
+import Tool from "../tool.js";
+import { CONSTANTS } from "../../constants/constants.js";
+import { getItemClass } from "../get-item-class.js";
+
+export default class Knife extends Tool {
+    // Text data
+    name = "Knife";
+    item = "Knife";
+    examineText = "A dangerous looking knife.";
+
+    // Other
+    cost = 6;
+
+    constructor(scene) {
+        super();
+        this.scene = scene;
+    }
+
+    async craft(item) {
+        console.log("Combining", this.name, item.name);
+        let className = "";
+        let numRequiredItems = 0;
+        let xpGiven = 0;
+        switch (item.name) {
+            case "Logs":
+                className = "NormalShortbow";
+                numRequiredItems = 50;
+                xpGiven = 50;
```

Comment:
This seems a little light given it requires 50 logs. Maybe have it at 5xp per log?

Reply:
I think it should be increased but 5x is too much since you can chop logs way faster than in the normal game. Maybe 2x?

Reply:
I'm thinking everything in rs clicker will be much faster than the real game. Might as well keep the xp ratios for logs from wc to fletching the same

Reply:
Created a draft PR here: https://github.com/CurtisGreen/rs-clicker/pull/7. I'll make the text color changes when you give me confirmation on the format